homework-classes/Repository.js

```diff
@@ -13,8 +13,47 @@ class Repository {
    * @param {HTMLElement} container The container element in which to render the repository.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.repository, null, 2) });
+    const repoTable = Util.createAndAppend('table', container);
+    const repoTableBody = Util.createAndAppend('tbody', repoTable);
+    const repoNameRow = this.addRepoInfoRow(repoTableBody, 'Repository', '');
+    Util.createAndAppend('a', repoNameRow.lastChild, {
+      href: this.repository.html_url,
+      target: '_blank',
+      text: this.repository.name,
+    });
+    this.addRepoInfoRow(repoTableBody, 'Description', this.repository.description);
+    this.addRepoInfoRow(
+      repoTableBody,
+      'Created At',
+      new Date(this.repository.created_at).toLocaleString('en-GB'),
+    );
+    this.addRepoInfoRow(
+      repoTableBody,
+      'Updated At',
+      new Date(this.repository.updated_at).toLocaleString('en-GB'),
+    );
+    this.addRepoInfoRow(repoTableBody, 'Forks', this.repository.forks_count);
+    this.addRepoInfoRow(repoTableBody, 'Watchers', this.repository.watchers_count);
+  }
+
+  /**
+   * Returns a row for repository container with desired label and content
+   *
+   * @param {HTMLElement} parent Parent tbody element
+   * @param {string} label Label text
+   * @param {string} content Content text
+   * @returns Repo info row
+   */
+  addRepoInfoRow(parent, label, content) {
```

Comment:
It seems you have grasped the concept of JSDoc, as applied to your own method.

Reply:
Thanks. It is a very handy tool for generating documentation.